content/en/user/moderating.md

```diff
@@ -35,6 +35,7 @@ Choose where the filter will be applied:
 * Notifications = matching notifications will not be shown
 * Public timelines = matching statuses will not appear in local/federated timelines
 * Conversations = matching statuses will be hidden in threads and detailed views
+* Profiles = matching profiles will be hidden
```

Comment:
`matching profiles will be hidden` -> `matching statuses will be hidden in profile views`?

Reply:
that would be a better way of phrasing it.